tool/tsh/tsh.go

```diff
@@ -530,6 +532,7 @@ func Run(ctx context.Context, args []string, opts ...cliOption) error {
 	daemon := app.Command("daemon", "Daemon is the tsh daemon service").Hidden()
 	daemonStart := daemon.Command("start", "Starts tsh daemon service").Hidden()
 	daemonStart.Flag("addr", "Addr is the daemon listening address.").StringVar(&cf.DaemonAddr)
+	daemonStart.Flag("certsDir", "Directory where certificates for sercure communication are stored").StringVar(&cf.DaemonCertsDir)
```

Comment:
```suggestion
	daemonStart.Flag("certs-dir", "Directory where certificates for sercure communication are stored").StringVar(&cf.DaemonCertsDir)
```

Flags should all be lowercase.

Reply:
Done